### ISSUE TITLE
something weird and poor results training on kitti

### ISSUE THREAD

**Issue by didibaba** [xingyizhou/CenterNet#81]
![Capture3](https://user-images.githubusercontent.com/12743672/57917210-cc6c0600-78c6-11e9-9502-ce01e41defd6.jpg)

I have trained 3dop model on kitti dataset and got really poor results as is shown in the image above. Also there was a weird error while testing.  I have four NV M40 GPUs and I enlarged the batch_size.

```
cd src
# train
python main.py ddd --exp_id 3dop --dataset kitti --kitti_split 3dop --batch_size 64 --master_batch 16 --num_epochs 70 --lr_step 45,60 --gpus 0,1,2,3

# test
python test.py ddd --exp_id 3dop --dataset kitti --kitti_split 3dop --resume
cd ..
```

Could you please tell me what is the problem?

**Comment by xingyizhou** [comment]
The metric for 3d detection we used is `car_ground`. What is the wired error?

**Comment by didibaba** [comment]
This is the testing results I have obtained. Is this correct? 

> car_detection AP: 97.357979 87.940445 79.244270
> car_orientation AP: 94.236603 84.230827 75.612167
> pedestrian_detection AP: 69.131393 60.497147 52.087666
> pedestrian_orientation AP: 53.802502 46.410378 40.152214
> cyclist_detection AP: 64.177635 40.516460 39.982079
> cyclist_orientation AP: 56.210171 35.506565 34.864361
> car_detection_ground AP: 30.589405 29.040533 25.690094
> pedestrian_detection_ground AP: 19.019531 15.540272 14.692955
> cyclist_detection_ground AP: 20.639729 15.275107 15.173965
> car_detection_3d AP: 18.339911 18.313784 16.163464
> pedestrian_detection_3d AP: 18.452168 15.117846 14.200698
> cyclist_detection_3d AP: 20.270189 15.025520 15.037395

**Comment by didibaba** [comment]
![my](https://user-images.githubusercontent.com/12743672/58065053-eb181880-7bb6-11e9-8ff8-9c430d02638e.jpg)

![ddd](https://user-images.githubusercontent.com/12743672/58065012-d6d41b80-7bb6-11e9-8b31-247d0822ff45.jpg)

These are the results testing on the original model and my model.

**Comment by xingyizhou** [comment]
Both the numbers and the qualitative results look good to me. We observed up to 7 BEV AP jittering for KITTI. We also reported the std of kitti results in the paper.

**Comment by abhigoku10** [comment]
@didibaba  hello just have few queries , did u train on kitti Birds eye view image ?? if so what is the mAP you have received and the fps on the given feature backbone , how much time did it take to train on GPU
